db/__tests__/seed.test.js

```diff
@@ -88,6 +87,50 @@ test('GET 400: responds with 400 error message if id is entered in an invalid fo
         expect(msg).toBe('Bad Request')
     })
 })
+test('PATCH 200: updates votes key of article and returns updated article', () => {
+    return request(app)
+    .patch('/api/articles/1')
+    .send({ inc_votes: 1 })
+    .expect(200)
+    .then(({ body }) => {
+        const { article } = body
+            expect(article).toMatchObject(expect.objectContaining(
```

Comment:
If you're using .toMatchObject you don't also need .objectContaining, you could use .toEqual(expect.objectContaining({})) as an alternative